src/sample_from_set.jl

```diff
@@ -0,0 +1,100 @@
+function load_distributions_sample_from_set()
```

Comment:
Maybe the file name `sample_from_set.jl` could be shortened to `sample.jl`. @mforets?

Reply:
There is also a file `samples.jl` should I merge this to files?

Reply:
Yes!